server/plugins/api/Slack.ts

```diff
@@ -0,0 +1,33 @@
+import { APIReturn } from "./APIReturn"
+import { GenericNotifier } from "./GenericNotifier"
+import { GenericConfig } from "./GenericConfig"
+
+interface Config extends GenericConfig {
+  groups?: string[],
+  channel?: string,
```

Comment:
maybe merge `channel` & `channels`?

Reply:
can do merging in config, kinda vague rn